testutil/tls.go

```diff
@@ -57,61 +54,81 @@ func (p *pki) TLSMaxVersion() string {
 }
 
 func (p *pki) ReadClientCert() string {
-	return readCertificate(p.ClientCertPath())
+	return tls.ReadCertificate(p.ClientCertPath())
 }
 
 func (p *pki) ClientCertPath() string {
 	return path.Join(p.path, "clientcert.pem")
 }
 
 func (p *pki) ReadClientKey() string {
-	return readCertificate(p.ClientKeyPath())
+	return tls.ReadKey(p.ClientKeyPath())
 }
 
 func (p *pki) ClientKeyPath() string {
 	return path.Join(p.path, "clientkey.pem")
 }
 
+func (p *pki) ReadClientCertAndKey() string {
+	return tls.ReadKey(p.ClientCertAndKeyPath())
```

Comment:
Is ReadClientCertAndKey() used anywhere? If not please remove it. If so, doesn't it need to call tls.ReadCertificate() here too?